v4/source/permissions.yaml

```diff
@@ -0,0 +1,31 @@
+  /permissions/ancillary:
+    get:
+      tags:
+        - permissions
+      summary: Return all system console subsection ancillary permissions
```

Comment:
This actually returns all the system console subsection permissions appended to the parameters being passed in.